HunterPie/Features/Native/IPCInjectorInitializer.cs

```diff
@@ -15,9 +15,6 @@ internal class IPCInjectorInitializer : IContextInitializer
 
         public void Initialize(Context context)
         {
-            if (context is not MHRContext)
-                return;
-
```

Comment:
The DLL **should not** be injected into MHR Demo

Reply:
You know? I'm thinking of moving the `Process.Memory.Inject` calls to another abstract function in `Context` 😂

Reply:
I think that would fit the internal `IContext` idea then